nakedret_test.go

```diff
@@ -65,3 +82,25 @@ func TestCheckNakedReturns(t *testing.T) {
 		})
 	}
 }
+
+func TestAll(t *testing.T) {
+	wd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Failed to get wd: %s", err)
+	}
+
+	testdata := filepath.Join(wd, "testdata")
+	log.Printf("Running tests on %s", testdata)
+	analysistest.Run(t, testdata, NakedReturnAnalyzer(0), "x")
+}
+
+func TestAllFixes(t *testing.T) {
+	wd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Failed to get wd: %s", err)
+	}
+
+	testdata := filepath.Join(wd, "testdata")
+	log.Printf("Running tests on %s", testdata)
```

Comment:
Maybe `t.Log` instead of `log.Printf`?

Btw., do we really need this log here?

Reply:
removed, not really needed